.eslintrc.json

```diff
@@ -14,7 +14,8 @@
         "inject": false,
         "module": false,
         "Sk": false,
-        "tie": false
+        "tie": false,
+        "tieBasic": false
```

Comment:
It depends on how you plan to use the module. To me, "tieCommon" means stuff that's shared between the various pages; "tieConfig" just means stuff used in configuration (so no utility functions, libraries, etc.). I lean towards the latter so that it doesn't become a grab-bag, but only mildly. What are your thoughts?

Might also be worth discussing with @theresa-lee (in terms of figuring out what sorts of things we might extract out from various pages in the future).